src/platform/EFR32/CHIPDevicePlatformConfig.h

```diff
@@ -48,17 +48,15 @@
 
 #define CHIP_DEVICE_CONFIG_ENABLE_CHIP_TIME_SERVICE_TIME_SYNC 0
 
-#if defined(RS911X_WIFI)
+#if defined(RS911X_WIFI) || defined(WF200_WIFI)
```

Comment:
i m sure this pattern repeats often 
can't we think of a #define SI_MODULE_WIFI to gathering both modules ?

Reply:
using `#if defined(SL_WIFI)`